base/libgit2/remote.jl

```diff
@@ -53,6 +87,12 @@ function fetch_refspecs(rmt::GitRemote)
     res
 end
 
+"""
+    push_refspecs(rmt::GitRemote) -> Vector{String}
+
+Get the *push* refspecs for the specified `rmt`. These refspecs contain
```

Comment:
I suppose since the fetch and push refspecs may be different. I should bold fetch for consistency as well.